nginx.conf.sigil

```diff
@@ -0,0 +1,227 @@
+{{ range $port_map := .PROXY_PORT_MAP | split " " }}
+{{ $port_map_list := $port_map | split ":" }}
+{{ $scheme := index $port_map_list 0 }}
+{{ $listen_port := index $port_map_list 1 }}
+{{ $upstream_port := index $port_map_list 2 }}
+
+limit_req_zone $binary_remote_addr zone=one:10m rate=1r/s;
```

Comment:
Given these are global to nginx, can I suggest more descriptive names?

e.g.

```
limit_req_zone $binary_remote_addr zone=jobserver_login:10m rate=1r/s;
limit_req_zone $binary_remote_addr zone=jobserver_default:10m rate=20r/s;
```

Reply:
Hmm, AIUI, we can't really reuse limit zones, or else you'd be linking the limits across multiple locations?

so if another url use `one` like /login does, I think that those limits would be shared?

Reply:
ah, yeah, you're right, the two urls would share the limit, let's not do that.